ex7.15_NumPy_fill_three_dimension_array.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def three_dimension():
+    """
+    function that create a three-dimension array with shape (300,400,5) and set to a variable. Fill the
+    array elements with values using unsigned integer (0 to 255).
+    :return:
+    """
+    np.random.seed(32)
```

Comment:
Why is the seed inside the function. Every time the function is called you will receive the same value. If this is part of a larger program you will continue udating the seed causing the rest of the program not be "random".